aiperf/post_processors/base_metrics_processor.py

```
@@ -35,6 +36,22 @@ def get_filters(self) -> tuple[MetricFlags, MetricFlags]:
+        """
+        if not self.user_config.input.goodput:
+            return
+        if GOOD_REQUEST_COUNT_TAG not in applicable_tags:
+            return
+
+        try:
+            MetricRegistry.get_class(GOOD_REQUEST_COUNT_TAG).set_slos(
+                self.user_config.input.goodput
+            )
+        except ValueError as e:
+            raise RuntimeError(f"Invalid --goodput: {e}") from e
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)